tensorflow/contrib/verbs/verbs_server_lib.cc

```diff
@@ -103,6 +103,7 @@ Status VerbsServer::Start() {
           ThreadOptions(), "TF_verbs_service",
           [this] { verbs_service_->HandleRPCsLoop(); }));
       rdma_mgr_->SetupChannels();
```

Comment:
Can we assume all servers are up and running and connected after SetupChannels is finished ?